TTYC.Application/Products/GetProductsList/GetProductsListHandler.cs

```diff
@@ -14,10 +14,10 @@ public GetProductsListHandler(ApplicationDbContext dbContext)
             this.dbContext = dbContext;
         }
 
-        public async Task<PagedList<Product>> Handle(GeStoresListQuery query, CancellationToken cancellationToken)
+        public async Task<PagedList<Product>> Handle(GetProductsListQuery query, CancellationToken cancellationToken)
         {
             var products = await PagedList<Product>.ToPagedListAsync(
-                dbContext.Products,
+                dbContext.Products.Where(x => x.IsActive == true),
```

Comment:
IsActive is already a bool, you can make it like this dbContext.Products.Where(x => x.IsActive)